man/man5/zfs-module-parameters.5

```diff
@@ -194,11 +194,29 @@ Default value: \fB2\fR.
 .ad
 .RS 12n
 Scales \fBl2arc_headroom\fR by this percentage when L2ARC contents are being
-successfully compressed before writing. A value of 100 disables this feature.
+successfully compressed before writing. A value of \fB100\fR disables this
+feature.
 .sp
 Default value: \fB200\fR%.
 .RE
 
+.sp
+.ne 2
+.na
+\fBl2arc_trim_ahead\fR (ulong)
+.ad
+.RS 12n
+Trims ahead of the current evict hand on L2ARC devices by this percentage of
+write size if we have filled the device. It has a minimum value of 64MB. If
```

Comment:
I was initially confused by the phrase "It has a minimum value of 64MB.", since the previous sentence stated this was a percentage.  How about rewording it slightly and moving it after the following sentence, something like:

"Trims ahead of the current evict hand on L2ARC devices by this percentage of
write size if we have filled the device.  If set to \fB100\fR we TRIM twice the space
required to accommodate upcoming writes.  A minimum of 64MB will be trimmed.
It also enables TRIM ..."
```

nit: fixed spelling of accommodate in original text.